src/guide/jbs-jdk-bug-system.md

```diff
@@ -103,7 +103,7 @@ Reasons why this would be the case include the fact that the source has been rem
 Do not:
 
 - use the label to indicate that a bug is not relevant to an earlier release, for example<br>[Affects Version]{.jbs-field}: [11.0.20]{.jbs-value}, [17]{.jbs-value}<br> the label [8-na]{.jbs-label} would not be needed - as it doesn't have a JDK 8 release, or earlier, value in the [Affects Versions]{.jbs-field}, it is not relevant to JDK 8.
-- add multiple [-na]{.jbs-label} labels: you only need one, for don't add both [9-na]{.jbs-label} and [11-na]{.jbs-label} — [9-na]{.jbs-label} implies all following releases therefore [11-na]{.jbs-label}, [17-na]{.jbs-label} etc. is not needed.
+- add multiple [-na]{.jbs-label} labels: you only need one, for example don't add both [9-na]{.jbs-label} and [11-na]{.jbs-label} — [9-na]{.jbs-label} implies all following releases therefore [11-na]{.jbs-label}, or [17-na]{.jbs-label} etc. are not needed.
```

Comment:
There are only two items between “or”, without the comma it looks better to me.

Reply:
ok thanks